packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -0,0 +1,206 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "@material/typography/mixins";
+
+$mdc-grid-list-tile-width: 200px !default;
+$mdc-grid-list-tile-secondary-padding: 16px;
+$mdc-grid-list-tile-secondary-padding-narrow: 8px;
+$mdc-grid-list-oneline-cap-secondary-height: 48px;
+$mdc-grid-list-twoline-cap-secondary-height: 68px;
+$mdc-grid-list-tile-secondary-icon-size: 24px;
+
+@mixin mdc-grid-list-tile-aspect($width-height-ratio) {
+  .mdc-grid-tile__primary {
+    padding-bottom: calc(100% / #{$width-height-ratio});
+  }
+}
+
+@mixin mdc-grid-list-tile-gutter($gutter-width) {
+  .mdc-grid-tile {
+    margin: $gutter-width/2 0;
+    padding: 0 $gutter-width/2;
+
+    &__secondary {
+      left: $gutter-width/2;
+      width: calc(100% - #{$gutter-width});
+    }
+  }
+
+  .mdc-grid-list__tiles {
+    margin: $gutter-width/2 auto;
+  }
+}
+
+// postcss-bem-linter: define grid-list
+.mdc-grid-list {
+  @include mdc-grid-list-tile-aspect(1);
+  @include mdc-grid-list-tile-gutter(4px);
+
+  // stylelint-disable plugin/selector-bem-pattern
+  // We need to disable linter here because we are nesting grid-tile under
+  // grid list here.
+  .mdc-grid-tile {
+    &__secondary {
+      bottom: 0;
+      height: $mdc-grid-list-oneline-cap-secondary-height;
+      padding: $mdc-grid-list-tile-secondary-padding;
+    }
+
+    &__icon {
+      font-size: 0;
+    }
+  }
+  // stylelint-enable plugin/selector-bem-pattern
+
+  &__tiles {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.mdc-grid-list--tile-gutter-1 {
+  @include mdc-grid-list-tile-gutter(1px);
+}
+
+.mdc-grid-list--tile-aspect-16x9 {
+  @include mdc-grid-list-tile-aspect(16 / 9);
+}
+
+.mdc-grid-list--tile-aspect-3x2 {
+  @include mdc-grid-list-tile-aspect(3 / 2);
+}
+
+.mdc-grid-list--tile-aspect-2x3 {
+  @include mdc-grid-list-tile-aspect(2 / 3);
+}
+
+.mdc-grid-list--tile-aspect-4x3 {
+  @include mdc-grid-list-tile-aspect(4 / 3);
+}
+
+.mdc-grid-list--tile-aspect-3x4 {
+  @include mdc-grid-list-tile-aspect(3 / 4);
+}
+
+// stylelint-disable plugin/selector-bem-pattern
+// We need to disable linter here because we are nesting grid-tile under grid list here.
+.mdc-grid-list--twoline-caption {
+  .mdc-grid-tile__secondary {
+    height: $mdc-grid-list-twoline-cap-secondary-height;
+  }
+}
+
+.mdc-grid-list--header-caption {
+  .mdc-grid-tile__secondary {
+    top: 0;
+    bottom: auto;
+  }
+}
+
+.mdc-grid-list--with-icon-align-start {
+  .mdc-grid-tile__secondary {
+    @include mdc-rtl-reflexive-property(padding, $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size, $mdc-grid-list-tile-secondary-padding-narrow, ".mdc-grid-list");
+
+    .mdc-grid-tile__icon {
```

Comment:
does this need to be nested under `.mdc-grid-tile__secondary` or can it be moved up one level? 

(same for below)